perllib/FixMyStreet/Cobrand/TfL.pm

```diff
@@ -80,4 +80,17 @@ sub admin_allow_user {
     return $user->from_body->name eq 'TfL';
 }
 
+sub must_have_2fa {
+    my ($self, $user) = @_;
+
+    require Net::Subnet;
+    my $ips = $self->feature('internal_ips');
+    my $is_internal_network = Net::Subnet::subnet_matcher(@$ips);
+
+    my $ip = $self->{c}->req->address;
+    return 0 if $is_internal_network->($ip);
+    return 1 if $user->from_body && $user->from_body->name eq 'TfL';
```

Comment:
We should probably return 1 for superusers too.

Reply:
I'd hope they'd all already logged into .com, but sure :)